pkg/kubelet/kubelet_getters.go

```diff
@@ -262,23 +262,23 @@ func (kl *Kubelet) GetPodCgroupRoot() string {
 	return kl.containerManager.GetPodCgroupRoot()
 }
 
-// GetHostIP returns host IP or nil in case of error.
-func (kl *Kubelet) GetHostIP() (net.IP, error) {
+// GetHostIPs returns host IPs or nil in case of error.
+func (kl *Kubelet) GetHostIPs() ([]net.IP, error) {
 	node, err := kl.GetNode()
 	if err != nil {
 		return nil, fmt.Errorf("cannot get node: %v", err)
 	}
-	return utilnode.GetNodeHostIP(node)
+	return utilnode.GetNodeHostIPs(node)
```

Comment:
DO we now need to pluralize pod.status.hostIP ?

Reply:
there is an open issue about that https://github.com/kubernetes/kubernetes/issues/85443
It seems so

Reply:
I mentioned that in the KEP; it's not a critical fix, since for the most part, every client in a dual-stack cluster ought to be able to reach either node IP, so having easy API access to one of them but not the other should not be fatal. But probably should be fixed for beta.